core/standalone/build.gradle

```diff
@@ -16,20 +16,15 @@
  */
 
 plugins {
+    id 'maven'
+    id 'org.scoverage'
     id 'org.springframework.boot' version '2.1.6.RELEASE'
-    id "com.gorylenko.gradle-git-properties" version "2.0.0"
     id 'scala'
+    id 'com.gorylenko.gradle-git-properties' version '2.0.0'
```

Comment:
While we are at this lets move all plugin versions to `settings.gradle` such that all versions are at one place. See https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_version_management for details.

Something like below in `settings.gradle`

```gradle
pluginManagement {
    plugins {
        id 'cz.alenkacz.gradle.scalafmt' version '1.8.0'
        id 'org.scoverage' version '3.2.0'
        id 'com.gorylenko.gradle-git-properties' version '2.2.0'
        id 'org.springframework.boot' version '2.2.1.RELEASE'
    }
}
```

Reply:
This seems to only work in gradle 6. The gradle 5 documentation doesn't mention anything about it and the bump to gradle 6 is still pending.

Reply:
I am currently using it with 5.6.4 and it seems to be [documented in a different place][1] for 5.x series

[1]: https://docs.gradle.org/5.6.4/userguide/plugins.html#sec:plugin_version_management

Reply:
The gradle wrapper is setup for 5.5.1 though, which doesn't seem to have it: https://docs.gradle.org/5.5.1/userguide/plugins.html#sec:plugin_version_management.

I see your point in doing it, but I'd rather punt on it until I can make Gradle 6 work properly. Seems orthogonal to this change anyway.

Reply:
Ah missed that. So it got introduced in between 5.x series and not supported in version we use in OpenWhisk currently. Makes sense to take it as part of Gradle 6 upgrade #4734